doc/asciidoc/cloud/index.adoc

```diff
@@ -0,0 +1,28 @@
+
+== Confluent Cloud
+
+Configuring a connection to a Confluent Cloud instance should follow
+link:https://docs.confluent.io/current/cloud/using/config-client.html#java-client[Confluent's Java Client]
+configuration advice, and the advice in <<_kafka_settings, Kafka Settings>> section.
+At a minimum, to configure this, you will need:
+
+* `BOOTSTRAP_SERVER_URL`
+* `API-KEY`
+* `API-SECRET`
+
+More specifically the plugin has to be configured as follow:
+
+.neo4j.conf
+[source,ini]
+----
+kafka.bootstrap.servers: ${BOOTSTRAP_SERVER_URL}
```

Comment:
this seems to be mixing neo4j.conf format with docker format.  if it's in neo4j.conf, you use = not : .  And if it's in docker format you need NEO4J_kafka_bootstrap_servers, etc.

Reply:
You're absolutely right! I'll fix it and i'll update the PR. Thanks for pointing that out